catalog/pages/tabs/index.md

```diff
@@ -77,3 +81,37 @@ responsive: true
 </TabsProvider>
 </div>
 ```
+
+### Tabs with TabsProvider and TabsConsumer components on a dark background
+
+```react
+responsive: true
+---
+<div style={{
+  padding: '16px',
+  backgroundColor: themes.tm.brand
+}}>
+<TabsProvider>
+    <TabsConsumer>
+      {({ tabIndex, onChangeTabIndex }) => (
+        <React.Fragment>
+        <Tabs
+          items={['first tab', 'second tab', 'third tab', 'fourth tab', 'fifth tab', 'sixth tab']}
+          itemsProps={[{'data-customInfo': 'someInfo_0'}, {'data-customInfo': 'someInfo_1'}, {'data-customInfo': 'someInfo_2'}]}
```

Comment:
The `data-customInfo` causes a critical error log because it's getting rendering in the DOM and it isn't a valid attribute.  I think we should change it to `data-custom-info`.

Reply:
Good eye, I will update these